subsys/bluetooth/audio/shell/bap.c

```diff
@@ -675,6 +675,71 @@ static int set_metadata(struct bt_audio_codec_cfg *codec_cfg, const char *meta_s
 }
 
 #if defined(CONFIG_BT_BAP_UNICAST_CLIENT)
+int bap_ac_create_unicast_group(const struct bap_unicast_ac_param *param,
+				struct shell_stream *snk_uni_streams[], size_t snk_cnt,
+				struct shell_stream *src_uni_streams[], size_t src_cnt)
+{
+	struct bt_bap_unicast_group_stream_param snk_group_stream_params[BAP_UNICAST_AC_MAX_SNK] = {
+		0};
```

Comment:
not an error - just looks strange (formatter?)

Reply:
It's whatever clang-format chose to do :D 

For something like this, I am unsure of how I would format it otherwise, except for maybe doing a `memset` instead of `= {0}`?

Reply:
it's fine with {0} imo